keylime/ima/ast.py

```diff
@@ -29,11 +29,15 @@
 
 
 def get_START_HASH(hash_alg: Hash) -> bytes:
-    return codecs.decode(b"0" * (hash_alg.get_size() // 4), "hex")
+    hsize = hash_alg.get_size()
+    assert hsize is not None
+    return codecs.decode(b"0" * (hsize // 4), "hex")
 
 
 def get_FF_HASH(hash_alg: Hash) -> bytes:
-    return codecs.decode(b"f" * (hash_alg.get_size() // 4), "hex")
+    hsize = hash_alg.get_size()
+    assert hsize is not None
```

Comment:
What is driving this that you add an assert here? Why would the hsize possibly be None?

Reply:
https://github.com/mdrocco/keylime/blob/314dd2c8de2eeae5209b688903678f959f41ec3c/keylime/common/algorithms.py#L48